src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -759,5 +763,90 @@ private static bool IsPotentialFilterCharacter(char c)
                 || char.IsNumber(c)
                 || c == '_';
         }
+
+        private static (int index, bool forceHardSelection) GetDefaultsMatch(
```

Comment:
so the term 'defaults' is really hard to understand in these contexts.  Why is it called a 'default'? what does 'default' mean?